src/components/ScrollZone.tsx

```diff
@@ -30,7 +28,8 @@ const ScrollZone = ({ leftHanded }: { leftHanded?: boolean } = {}) => {
       })}
       style={{
         transform: `translateY(calc(-${scrollTop / 4 + 400}px))`,
-        width: scrollZoneWidth,
+        // as per viewportStore.scrollZoneWidth = 0.39, but we just go with CSS-based calc
+        width: '39vmin',
```

Comment:
I don't recommend decoupling the width from `scrollZoneWidth`. That creates future risk that the values will diverge without us knowing. We want the code to enforce the tight coupling.

Reply:
yep, that's why i didn't really wanna do this :D
count this a question